### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by erichodges**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -64,7 +64,7 @@ GEM
     multi_json (1.10.1)
     nokogiri (1.6.4)
       mini_portile (~> 0.6.0)
-    pg (0.17.1)
+    pg (0.18.4)
```
Comment (tebba-von-mathenstein) [inline Gemfile.lock]: Hey -- our production version is locked on 0.17, I know have some dev issues with PG 0.17, but we're going to have to keep the PROD version static. Can you revert this, and we'll test your work on staging to make sure nothing you did is pg 0.17 specific.

File: app/views/quotes/index.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+      <%= time_ago_in_words(quote.created_at) + " ago" %>
```
Comment (tebba-von-mathenstein) [inline app/views/quotes/index.html.erb]: Can we revert this? We don't need to do the string concatenation, the html rendering for `ago` should work just fine :)

File: db/production.sql [hunk 3]
```diff
@@ -8,8 +8,8 @@ SET client_min_messages = warning;
 SET search_path = public, pg_catalog;
 
 COPY quotes (id, text, author, created_at, updated_at) FROM stdin;
-1	People who think they know everything are a great annoyance to those of us who do.	Isaac Asimov	\N	\N
-2	As a child my family's menu consisted of two choices: take it or leave it.	Buddy Hackett	\N	\N
+1	People who think they know everything are a great annoyance to those of us who do.	Isaac Asimov	2014-07-12 03:40:54.405448	2014-07-12 03:40:54.405448
+2	As a child my family's menu consisted of two choices: take it or leave it.	Buddy Hackett	2014-07-12 03:40:54.405448	2014-07-12 03:40:54.405448
```
Comment (tebba-von-mathenstein) [inline db/production.sql:12]: Nice catch -- these should definitely not have been null. Can you add a validator to the quote model that would reject null entries in the database? 

http://guides.rubyonrails.org/active_record_validations.html

Comment (tebba-von-mathenstein) [inline db/production.sql:12]: And write some tests to see if that validator is working :)

**Comment by tebba-von-mathenstein** [comment]
Thanks for getting this done, please see my comments in line :)

File: db/production.sql (hunk 3 above)
Comment (erichodges) [inline db/production.sql:12]: Hi Tyler, I added validation and tests for the validation.

**Comment by tebba-von-mathenstein** [comment]
Thanks for making those changes Eric. This looks great. I'm going to go ahead and ship this code and handle the merge on my end, so you're all set. 

:)

**Pull request closed without merging**